hack/util.sh

```diff
@@ -94,20 +97,26 @@ function start_os_server {
       --master-config=${MASTER_CONFIG_DIR}/master-config.yaml \
       --node-config=${NODE_CONFIG_DIR}/node-config.yaml \
       --loglevel=4 \
-      1>&2 2>"${BASETMPDIR}/openshift.log" &
+      1>&2 2>"${LOG_DIR}/openshift.log" &
```

Comment:
With the change I made, the openshift.log IS NOW in the logs dir.  It previously was at the base dir, a level above the logs dir, when BASETMPDIR was used.  Switching to LOG_DIR moved openshift.log to the logs dir.